config/environments/production.rb

```diff
@@ -69,15 +69,15 @@
   # Growstuff configuration
   config.action_mailer.default_url_options = { host: 'growstuff.org' }
 
-  config.action_mailer.smtp_settings = {
-      port: '587',
-      address: 'smtp.mandrillapp.com',
-      user_name: ENV['GROWSTUFF_MANDRILL_USERNAME'],
-      password: ENV['GROWSTUFF_MANDRILL_APIKEY'],
-      domain: 'heroku.com',
-      authentication: :plain
+  ActionMailer::Base.smtp_settings = {
+      port:                 ENV['SPARKPOST_SMTP_PORT'],
```

Comment:
We've been using the convention that all Growstuff-related environment variables (which should at least include our SparkPost username and password) should have the prefix GROWSTUFF_.

Also, these environment variables should be documented in config/environment.yml.example.